edits to n-3 PUFA section (#814)
greenelab/covid19-review: content/21.nutraceuticals.md

### n-3 PUFA

> One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs [@doi:10.1002/jmv.25707], which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).

Reviewer: ```suggestion
One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs [@doi:10.1002/jmv.25707], commonly referred to as omega-3 fatty acids, which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
```

> EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils (FO) or purified oils [@doi:10.1039/c9fo01742a].

Author: ```suggestion
EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils or purified oils [@doi:10.1039/c9fo01742a].
```

This and other evidence indicates that n-3 PUFAs may have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.1039/c9fo01742a; @doi:10.1038/nrcardio.2016.103]; for example, they have been found to influence antigen presentation and the production of CD4(+) Th1 cells, among other relevant effects [@doi:10.1016/j.ejphar.2015.03.100].
Interest has also arisen as to whether nutritional status related to n-3 PUFAs can also affect inflammation associated with severe disease, such as ARDS or sepsis [@doi:10.1007/s11745-016-4129-x; @doi:10.1016/j.nut.2019.02.016].
ARDS and sepsis hold particular concern in the treatment of severe COVID-19;  an analysis of 82 deceased COVID-19 patients in Wuhan during January to February 2020 reported that respiratory failure (associated with ARDS) was the cause of death in 69.5% of cases, and sepsis or multi-organ failure accounted for 28.0% of deaths [@doi:10.1371/journal.pone.0235458].
Research in ARDS prior to current pandemic suggests that n-3 PUFAs may hold some therapeutic potential.

> One study randomized 16 consecutive ARDS patients to receive either an FO-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design [@doi:10.1186/1743-7075-8-22].

Reviewer: ```suggestion
One study randomized 16 consecutive ARDS patients to receive either a fish oil-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design [@doi:10.1186/1743-7075-8-22].
```

> They reported a statistically significant reduction in leukotriene B4 levels in the group receiving the FO-enriched emulsion, suggesting that the FO supplementation may have reduced inflammation.

Reviewer: ```suggestion
They reported a statistically significant reduction in leukotriene B4 levels in the group receiving the fish oil-enriched emulsion, suggesting that the FO supplementation may have reduced inflammation.
```

However, they also reported that most of their tests were not statistically significant, and therefore it seems that additional research using larger sample sizes is required.
A recent meta-analysis of 10 randomized controlled trials (RCTs) examining the effects of omega-3 fatty acids on ARDS patients did not find evidence of any effect on mortality, although the effect on secondary outcomes could not be determined due to a low quality of evidence [@doi:10.4187/respcare.06965].

> However, another meta-analysis that examined 24 RCTs studying the effects of omega-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when omega-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions [@doi:10.1186/s12937-018-0356-8].

Reviewer: ```suggestion
However, another meta-analysis that examined 24 RCTs studying the effects of n-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when n-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions [@doi:10.1186/s12937-018-0356-8].
```

> Therefore, despite theoretical support for an immunomodulatory effect of n-3 PUFAs in COVID-19, evidence from existing RCTs is insufficient to determine whether supplementation offers an advantage in a clinical setting that would be relevant to COVID-19.

Reviewer: I really like this section, good on you @rando2!

Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration [@doi:10.1016/j.jacc.2020.04.031; @doi:10.3389/fnut.2020.583080].
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19.
In particular, concerns have been raised within the medical community about using investigational therapeutics on COVID-19 patients who are already on antiplatelet therapies due to pre-existing comorbidities because the introduction of such therapeutics could lead to issues with dosing and drug choice and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].

> In such cases, dietary sources of n-3 fatty acids or other nutraceuticals with antiplatelet activities could hold particular value for reducing the risk of thrombotic complications in patients already receiving pharmaceutical antiplatelet therapies. <!--TO DO: is this being explored/tested?-->

Reviewer: I would say that this is being explored in the literature, but I haven't seen any evidence of trials looking at platelet activities as a primary or secondary endpoint. Its a labour intensive trial if looking beyond the standard prothrombin time etc.

Reviewer: As you see below, they are mainly interested in the anti-inflammatory potential.
